web/src/hooks/redux.js

```diff
@@ -16,10 +16,23 @@ export function useCurrentZoneHistory() {
   return useMemo(() => histories[zoneId] || null, [histories, zoneId]);
 }
 
+export function useCurrentDatetimes() {
+  // TODO: should use V5 state here and v5 should tell the state datetimes
+  const histories = useSelector((state) => state.data.histories);
+  if (histories && Object.keys(histories).length) {
+    return histories[Object.keys(histories)[0]].map((h) => moment(h.stateDatetime).toDate());
+  } else {
+    return [];
+  }
+}
```

Comment:
This function assumes all histories have the same datetimes. As the comment suggests, V5/state should keep track of the relevant datetimes instead of the individual zones

I don't believe we have any cases where it makes sense for some zones to have a different set of datetimes than another.